packages/maskbook/src/database/Persona/Persona.db.ts

```diff
@@ -536,20 +536,21 @@ export async function queryRelationsPagedDB(
         if (cursor.value.linked !== linked.toText()) continue
         if (cursor.value.network !== options.network) continue
 
-        if (
-            firstRecord &&
-            options.after &&
-            options.after.linked.toText() !== cursor?.value.linked &&
-            options.after.profile.toText() !== cursor?.value.profile
-        ) {
+        if (firstRecord && options.after && options.after.profile.toText() !== cursor?.value.profile) {
```

Comment:
```suggestion
        if (firstRecord && options.after?.profile.toText() !== cursor?.value.profile) {
```

Reply:
This change does not determine if `options.after ` is not empty